src/adminserver/systemcfg/systemcfg.go

```diff
@@ -152,6 +152,12 @@ var (
 	repeatLoadEnvs = map[string]interface{}{
 		common.ExtEndpoint:   "EXT_ENDPOINT",
 		common.MySQLPassword: "MYSQL_PWD",
+		common.MySQLUsername: "MYSQL_USR",
+		common.MySQLDatabase: "MYSQL_DATABASE",
+		common.MySQLPort: &parser{
```

Comment:
Also needs MySQLHost?

Reply:
Yes, this is the same issue as Clair. since we allow user to use third part DB.